modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -630,6 +674,57 @@ public CompletableFuture<Void> start() {
                 TimeUnit.MILLISECONDS
         );
 
+        scheduledTableLeaseUpdateExecutor.scheduleAtFixedRate(() -> {
+                    for (Map.Entry<ZonePartitionId, Set<ReplicationGroupId>> entry : zonePartIdToTablePartId.entrySet()) {
+                        ZonePartitionId repGrp = entry.getKey();
+
+                        ReplicaMeta meta = placementDriver.getLeaseMeta(repGrp);
```

Comment:
Is `ReplicaMeta` == `Lease` in fact? What difference between Lease and Lease Meta? It's sounds ambiguous

Reply:
Lease is an internal abstraction in the placement driver, but Replica Meta is an interface available to the side component.